babi/screen.py

```diff
@@ -417,6 +417,10 @@ def replace(self) -> None:
                 'replace with', history='replace', allow_empty=True,
             )
             if response is not PromptResult.CANCELLED:
+                if response != '':
+                    # escaping last character example backslash '\'
+                    r = response[-1].encode('unicode-escape').decode()
+                    response = response[:-1] + r
```

Comment:
easier is `response.endswith('\\')` -- though this should be an error (invalid replacement)

Reply:
Escaping all backslashes in the replace string caused some of the replace_tests to fail.  The tests that failed: test_replace_with_multiple_newline, test_replace_with_newline, and test_replace_all.  Not sure if I should modify the tests 
I decided try a different approach.  Using a try and except block to catch the error and do a statues update 'invalid replace text.'  I found a difference between python3.6 and 3.7.  For example replace with: 'a\da' and 'a\pa' will not crash on 3.6 but will crash on 3.7.  Another example replace with: 'a\ga' will crash on both 3.6 and 3.7.
The try except block will is not a fix but will keep babi from crashing.